Obedient-Lobsters/WorkforceManagement
# Addition of database creation and seeding sql

# Description
This PR includes the addition of a file called ```BangazonDatabaseCreateSeed.sql``` that contains the code to create a database and seed it with the information needed for this project.
 Fixes # 23
 ## Type of change
 # Testing Instructions

Clone the Obedient Lobsters WorkforceManagement repo onto your machine. ```cd``` into that directory and open the project in Visual Studio Code.
Make sure the database is built on your local machine using the contents of ```BangazonDatabaseSeedBuild``` in the project root directory (see database setup above for instructions).
Link Visual Studio Code to that database by going to ```View``` and selecting the ```SQL Server Object Explorer```. Open that up and press the ```add SQL server``` button (looks like a column with a green plus). Then select ```local``` and pick the option that matches your local server.

If all went correctly, your database should be connected, and you can then run the project.
To run the project, press the green "play" triangle that is above the code editor, roughly in the middle.
Check to make sure that the project still builds and runs.
 # Checklist:
